docs/misc/ide-configuration.md

```diff
@@ -0,0 +1,13 @@
+# IDE Configuration
```

Comment:
This feels a bit too off-topic to store here. There are many *many* different potential "IDE" setups out there. It's likely that contributors will already have a preferred way of writing/navigating source code.

Reply:
@jlebon In this case, I will remove this altogether. Do you use an IDE, and if so what it is? and how do you configure all the include paths?

Thanks! :smile:

Reply:
> Do you use an IDE, and if so what it is?

It depends on your definition of "IDE" :)
I use a combination of tmux + vim + universal-ctags + cscope (for cross-project definitions).

> how do you configure all the include paths?

In vim, you can press `gf` to open filenames under your cursor. I also use Command-T and sometimes NERDTree for arbitrary file exploration.

If you mean compiler flags, I just run `make` in a separate terminal. Though you can set up vim to do this with the `:make` command and e.g. navigate from error to error.